datasets/GM0362_amstelveen/GM0362_amstelveen.derived.ad

```diff
@@ -134,45 +134,45 @@
 - electric_vehicle_profile_5_share = 0.0
 - solar_pv_profile_1_share = 1.0
 - solar_pv_profile_2_share = 0.0
-- annual_infrastructure_cost_gas = 8.236905182997704
+- annual_infrastructure_cost_gas = 8.236905182997703e-06
```

Comment:
This value has become a million times smaller. Is that correct?

Reply:
This was fixed during the NL2019 update